src/Components/HomePage/Homepage.js

```diff
@@ -22,7 +22,7 @@ export const Homepage = (props) => {
             <Link
               id="About"
               className="link"
-              to="/about">
+              to="/">
```

Comment:
Did you change the path because we don't currently have anything for the About section? If this is the case, we should just delete the link altogether.